spring/src/main/java/io/seata/spring/annotation/GlobalTransactionalInterceptor.java

```diff
@@ -107,6 +136,10 @@ public Object invoke(final MethodInvocation methodInvocation) throws Throwable {
             final GlobalTransactional globalTransactionalAnnotation =
                 getAnnotation(method, targetClass, GlobalTransactional.class);
             final GlobalLock globalLockAnnotation = getAnnotation(method, targetClass, GlobalLock.class);
+            final boolean disable = ConfigurationFactory.getInstance()
```

Comment:
Why local variables?

Reply:
What happens if the disable configuration item does not exist?

Reply:
> What happens if the disable configuration item does not exist?

when not present, the default value will be used